src/scripts/main.js

```diff
@@ -1 +1,83 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+function createNotification(state, content) {
+  const notification = document.createElement('div');
+
+  body.appendChild(notification);
+  notification.setAttribute('data-qa', 'notification');
+  notification.textContent = content;
+
+  if (state === 'success') {
+    notification.classList.add('success');
+  } else {
+    notification.classList.add('warning');
+  }
+}
+
+function createFirstPromise() {
+  const resolver = (success, error) => {
```

Comment:
Fix all cases please
```suggestion
  const resolver = (resolve, reject) => {
```